app/src/main/java/com/github/jorgecastillo/kotlinandroid/KotlinArchitectureApp.kt

```diff
@@ -1,16 +1,27 @@
 package com.github.jorgecastillo.kotlinandroid
 
 import android.app.Application
-import kategory.Id
+import com.github.jorgecastillo.architecturecomponentssample.model.error.CharacterError
+import com.github.jorgecastillo.kotlinandroid.di.context.GetHeroesContext
+import com.github.jorgecastillo.kotlinandroid.functional.AsyncResult
+import com.github.jorgecastillo.kotlinandroid.functional.MonadControl
+import kategory.GlobalInstances
+import kategory.InstanceParametrizedType
 
 class KotlinArchitectureApp : Application() {
 
-  override fun onCreate() {
-    super.onCreate()
-    initGlobalInstances()
-  }
+    override fun onCreate() {
+        super.onCreate()
```

Comment:
Just in case we are not statically checking for correct formatting or auto apply formating using tools like `detekt`. I'm not using it here (at least not yet) so yep, you would need to download and apply the given code style manually. But do not worry for now, it's not that important to be honest.